csp/adapters/perspective.py

```diff
@@ -197,20 +206,29 @@ def create_table(self, name, limit=None, index=None):
 
     def _instantiate(self):
         set_threadpool_size(self._threadpool_size)
-
-        manager = PerspectiveManager()
-
-        thread = threading.Thread(target=perspective_thread, kwargs=dict(manager=manager))
+        if _PERSPECTIVE_3:
+            server = Server()
+            client = server.new_local_client()
+            thread = threading.Thread(target=perspective_thread, kwargs=dict(client=client))
+        else:
+            manager = PerspectiveManager()
+            thread = threading.Thread(target=perspective_thread, kwargs=dict(manager=manager))
         thread.daemon = True
         thread.start()
 
         for table_name, table in self._tables.items():
             schema = {
                 k: v.tstype.typ if not issubclass(v.tstype.typ, csp.Enum) else str for k, v in table.columns.items()
             }
-            ptable = Table(schema, limit=table.limit, index=table.index)
-            manager.host_table(table_name, ptable)
+            if _PERSPECTIVE_3:
```

Comment:
Think this block is the other way around, the code under `if _PERSPECTIVE_3` is the same as before but the code in the else block is the new API.

Reply:
How is this passing the dep tests?

Reply:
I don't think there is any test of the perspective adapter on its own

Reply:
I'll see if I can add a very basic one

Reply:
Done (and tested on 2.x/3.x) - note that it is very basic, but will catch syntax/import issues.